src/dams_mcda/WSM_graphs_test.R

```diff
@@ -443,7 +444,7 @@ colnames(Dam2ScoreTable) <- criteria_inputs
 
 # Millinocket/Dam 3 output table(s)
 
-Dam3RawTable <- setDT(Mill_DataMatrix)
+Dam3RawTable <- setDT(East/Mill_DataMatrix)
```

Comment:
this might not be right

Reply:
Thanks, I'm hoping to get to this momentarily.